backend/src/Prelude/Prelude.fs

```diff
@@ -26,6 +26,13 @@ let failwith = failwith
                            IsHidden = true)>]
 let printf = printf
 
+// is this useful?
+[<CompilerMessageAttribute("printfn is banned, use Prelude.print instead",
+                           0,
+                           IsError = true,
+                           IsHidden = true)>]
+let printfn = printfn
```

Comment:
misc - I noticed that while I couldn't `printf` when debugging, I could still use `printfn`